tests/sentry/tasks/test_assemble.py

```diff
@@ -186,25 +186,37 @@ def test_artifacts(self):
         blob1 = FileBlob.from_file(ContentFile(bundle_file))
         total_checksum = sha1(bundle_file).hexdigest()
 
-        assemble_artifacts(
-            org_id=self.organization.id,
-            version=self.release.version,
-            checksum=total_checksum,
-            chunks=[blob1.checksum],
-        )
-
-        status, details = get_assemble_status(
-            AssembleTask.ARTIFACTS, self.organization.id, total_checksum
-        )
-        assert status == ChunkFileState.OK
-        assert details is None
-
-        release_file = ReleaseFile.objects.get(
-            organization=self.organization, release=self.release, name="~/index.js", dist=None
-        )
-
-        assert release_file
-        assert release_file.file.headers == {"Sourcemap": "index.js.map"}
+        for has_release_archives in (True, False):
```

Comment:
Can you use `@pytest.mark.parametrize` here?

Reply:
Not as long as it's part of a `TestCase`, I'm afraid (see https://docs.pytest.org/en/latest/how-to/unittest.html?highlight=unittest#pytest-features-in-unittest-testcase-subclasses).